src/Modules/ModParser.lua

```diff
@@ -3172,7 +3172,8 @@ local specialModList = {
 		mod("ShrineBuff", "LIST", { mod = mod("Life", "INC", 20) }),
 		mod("ShrineBuff", "LIST", { mod = mod("AreaOfEffect", "INC", 20) })
 	},
-	-- 
+	["you count as on low life while at (%d+)%% of maximum life or below"] = function(num) return { mod("LowLifePercentage", "BASE", num) } end,
+	["you count as on full life while at (%d+)%% of maximum life or above"] = function(num) return { mod("FullLifePercentage", "BASE", num) } end,
```

Comment:
Want to future-proof these and add 'life' as a parameter too?  Then we don't need a PR for if they ever add mana related mods like this.